### ISSUE TITLE
CSV and JSON reader

### ISSUE THREAD

**Issue by domoritz** [domoritz/arrow-wasm#12]
Support reading CSV files into a table. We currently only support IPC.

**Comment by TheNeuralBit** [comment]
Does arrow rust have a CSV and/or JSON reader? I only see [parquet](https://github.com/apache/arrow/tree/master/rust/parquet)

**Comment by domoritz** [comment]
Yes, they do: https://docs.rs/arrow/3.0.0/arrow/csv/reader/struct.Reader.html and https://docs.rs/arrow/3.0.0/arrow/json/reader/struct.Reader.html.
